frontend/src/store/Store.ts

```diff
@@ -134,6 +139,11 @@ export interface MeshToolbarState {
 }
 
 export interface MeshState {
+  // controlPlanes are pulled from the graph response and are used
+  // by other elements on the graph page. They are in the redux
+  // store to so that the other elements can stay in sync with
+  // with graph and we don't have to duplicate requests to get this info.
+  controlPlanes?: ControlPlane[];
```

Comment:
I'm not 100% sure it's necessary to increase redux state with the controlPlanes.  Are they really needed to trigger a state change?  Could they not be parsed out of the Mesh elements when needed?

Reply:
Not really, from an element (the selected control plane node) you can always work "up" to the controller or graph.   You could do something like:
```
const controlPlanes: ControlPlane[] | undefined = 
    this.state.controlPlaneNode?.getGraph().getNodes()?
        .filter(node => node.getData().infraType === MeshInfraType.ISTIOD)
        .map(node => node.getData().infraData);
 ```

Reply:
Ah that's good to know. I'll give that a shot.